src/CalcManager/winerror_cross_platform.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+
+typedef int32_t HRESULT;
```

Comment:
Should not be needed anymore

Reply:
@janisozaur Should this really be removed even though `SUCCEEDED` and `FAILED` depend on it?

Reply:
`HRESULT` is already converted to a standard type: https://github.com/Microsoft/calculator/pull/212 / https://github.com/Microsoft/calculator/pull/212/commits/15e7e8d0aa628329588171cd9514ee6a689d0f3c, so yes.